src/workloads/scale/MixedWorkloadsGenny.yml

```diff
@@ -81,278 +145,153 @@ Actors:
   - *quiesce
   - *nop
 
-- Name: Update_1
-  Type: CrudActor
-  Database: *dbname
-  ClientName: Update
-  Threads: 1
-  Phases:
-  - Phase: 0..1
-    Nop: true
-  - ExternalPhaseConfig:
-      Path: &PhasePath ../../phases/scale/MixPhases.yml
-      Key: UpdatePhase
-  - Phase: 3..10
-    Nop: true
+# Update Actors
+- ActorInstance:
+    Template: UpdateTemplate
+    Parameters:
```

Comment:
How about `TemplateName` and `TemplateParameters` to make it clear how these blocks are treated?

Reply:
Done.